### PR TITLE
More consistent internal naming

### PR DESCRIPTION
**Pull request by rainbreak**
Fixes some of the confusing naming done in #18

### CODE REVIEW

**Comment by rainbreak** [comment]
All tests passing.

File: contracts/auction.sol [hunk 1]
```diff
@@ -158,3 +158,41 @@ contract AssertiveAuction is Assertive, UsingAuctionDatabase {
...
+        assertBiddable(auctionlet_id, bid_how_much);
+        doBid(auctionlet_id, msg.sender, bid_how_much);
+        Bid(auctionlet_id);
+    }
+    // Allow parties to an auction to claim their take.
+    // If the auction has expired, individual auctionlet high bidders
+    // can claim their winnings.
+    function claim(uint auctionlet_id) {
+        assertClaimable(auctionlet_id);
+        doClaim(auctionlet_id);
```
Comment (jorisbontje) [inline contracts/auction.sol:178]: no `Claimed` event?

Comment (rainbreak) [inline contracts/auction.sol:178]: Events are minimal so far. We can expand as needed - I was leaving this to frontend input.

I didn't put `Claim` in as it doesn't cause an auction state change.

`claim` is only possible when the auctionlet has 'expired', after which point only the highest bidder can claim anyway.

File: contracts/types.sol [hunk 2]
```diff
@@ -29,3 +29,12 @@ contract AuctionType {
...
 
+contract AuctionFrontendType {
+    function bid(uint auctionlet_id, uint bid_how_much);
+    function claim(uint auctionlet_id);
+}
+
+contract SplittingAuctionFrontendType is AuctionFrontendType {
+    function bid(uint auctionlet_id, uint bid_how_much, uint quantity)
+        returns (uint new_id, uint split_id);
+}
```
Comment (rainbreak) [inline contracts/types.sol:40]: @jorisbontje: note frontend type defs ^

**Pull request merged**